gulpfile.js

```diff
@@ -42,4 +44,17 @@ gulp.task('serve', function() {
     }));
 });
 
+gulp.task('sass', function() {
+  gulp.src(['./sass/*.sass', './sass/*.scss'])
+      .on('error', function(e) {
+        handleError(e, this);
+      })
+      .pipe(sourcemaps.init())
+      .pipe(sass())
+      .pipe(sass({indentationSyntax: true})
+      .pipe(autoprefixer('last 2 versions'))
+      .pipe(sourcemaps.write('./'))
+      .pipe(gulp.dest('./css/'));
```

Comment:
If I could make a request, could this be made either:
```(gulp.dest('./dist'))``` or ```(gulp.dest('./dist/css'))```?
The reason I ask is because we already have the dist folder in the gitignore file, and this would keep the compiled css and sourcemaps out of version control.